ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.container.js

```diff
@@ -343,13 +351,14 @@ const mapStateToProps = (state, ownProps) => {
     maxValue,
     smartTransactionsOptInStatus,
     currentChainSupportsSmartTransactions,
+    isSmartTransaction: getSmartTransactionsEnabled(state),
```

Comment:
So, there is a slight difference between `getSmartTransactionsEnabled` and `getIsSmartTransaction`: https://github.com/MetaMask/metamask-extension/blob/develop/shared/modules/selectors/smart-transactions.ts#L121

You should likely do this: 
```
isSmartTransaction: getIsSmartTransaction(state)
```

Reply:
Since these changes aren't required for the fix, I've reverted the rename and just removed `isSmartTransaction` since it was never set and is no longer used.